src/clojure/clojure.ts

```diff
@@ -36,11 +40,26 @@ export const language = <ILanguage>{
 	tokenPostfix: '.clj',
 
 	brackets: [
+		{open: '[', close: ']', token: 'delimiter.square'},
 		{open: '(', close: ')', token: 'delimiter.parenthesis'},
 		{open: '{', close: '}', token: 'delimiter.curly'},
-		{open: '[', close: ']', token: 'delimiter.square'},
 	],
 
+	constants: ['true', 'false', 'nil'],
+
+	// delimiters: /[\\\[\]\s"#'(),;@^`{}~]|$/,
+
+	numbers: /^(?:[+\-]?\d+(?:(?:N|(?:[eE][+\-]?\d+))|(?:\.?\d*(?:M|(?:[eE][+\-]?\d+))?)|\/\d+|[xX][0-9a-fA-F]+|r[0-9a-zA-Z]+)?(?=[\\\[\]\s"#'(),;@^`{}~]|$))/,
+
+	characters: /^(?:\\(?:backspace|formfeed|newline|return|space|tab|o[0-7]{3}|u[0-9A-Fa-f]{4}|x[0-9A-Fa-f]{4}|.)?(?=[\\\[\]\s"(),;@^`{}~]|$))/,
+
+	escapes: /^\\(?:backspace|formfeed|newline|return|space|tab|o[0-7]{3}|u[0-9A-Fa-f]{4}|x[0-9A-Fa-f]{4}|.)?/,
```

Comment:
I don't really know a lot about Clojure, but shouldn't these symbolic names (`backspace|formefeed...`) be replaced with their JS regex values. i.e. `/^\\(?:b|f|n|t...`

https://clojure.org/api/cheatsheet
![image](https://user-images.githubusercontent.com/5047891/49637687-64a3bc00-fa06-11e8-934e-c835f2eeaef6.png)

Reply:
Thank you very much for taking the time to look at this, @alexandrudima ! That was an excellent catch! It was an overlook due to the lack of test coverage for strings.

I just pushed a commit (with test coverage) that handles escapes in strings correctly.

Cheers!